docs/index.html

```diff
@@ -649,11 +660,14 @@ <h1 class="mdl-mega-footer--heading">Previous Conferences</h1>
               <h1 class="mdl-mega-footer--heading">Committee</h1>
               <ul class="mdl-mega-footer--link-list">
                 <li><a href="http://gib.people.uic.edu/">Gib Bassett</a></li>
-                <li>Peter Carl</li>
+                <li><a href="https://www.linkedin.com/in/pbakosova/">Petra Bakosova</a></li>
```

Comment:
Please ask if that is the URL she wants listed.

Reply:
I defaulted to LinkedIn profiles where I had them, but I have also asked Petra, Soumya, and Justin what link they would prefer.

Reply:
We do have a mailing list so that these things are not clandestine and can be validated. I would use it.

Reply:
Emailed the list for discussion.